packages/notifi-react-card/lib/components/subscription/NotifiSubscriptionCard.tsx

```diff
@@ -77,10 +77,10 @@ export const NotifiSubscriptionCard: React.FC<
   const { params } = useNotifiClientContext();
 
   return (
-    <NotifiSubscriptionContextProvider {...params}>
-      <NotifiFormProvider>
+    <NotifiFormProvider>
+      <NotifiSubscriptionContextProvider {...params}>
```

Comment:
Reason:
The `render` and "initial fetch" methods need to consume NotifiFormProvider